manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "manifest_version": 2,
+    "name": "LiveAgent-remote-actions",
+    "version": "1.0",
+
+    "description": "AAAAAAAA",
+    "icons": {
+	"48": "icons/border-48.png"
+    },
+
+    "content_scripts": [
+	{
+	    "matches": ["<all_urls>"],
+	    "js": ["dist/bundle.js"]
+	}
+    ],
+    "permissions": [
+	"*://liveagent.com/"
```

Comment:
LiveAgent is not always on this domain.

Reply:
Fixed, now the extensions asks for `<all_urls>` and the filtering is done manually in the extension with regexes